model/simulationtests/autosize_hvac.rb

```diff
@@ -403,6 +403,7 @@ def make_tes_coil(model)
 # plhp_clg.autosizeReferenceCapacity
 plhp_clg.autosizeSourceSideReferenceFlowRate
 plhp_clg.autosizeLoadSideReferenceFlowRate
+plhp_clg.autosizeHeatRecoveryReferenceFlowRate
```

Comment:
Can't do that. you need to create a second HeatPumpPlantLoopEIRCooling so that it's an air source AND has a connection to the HeatRecoveryLoop (HR is only enabled for air source, and this one is a water source one)